doc/submitting-changes.xml

```diff
@@ -179,6 +179,10 @@ Additional information.
 
 <listitem>
 <para>Rebase you branch against current <command>master</command>.</para>
+<note>
+  <para>If the rebase fails it might be because the commit from <command>nixos-version</command> happened to be cherry-picked (not an ancestor of master). This is usually mentioned in the commit message.</para>
+  <para>A simple fix is to extract the commits (<command>git format-patch SHA_FIRST_COMMIT..HEAD --stdout > temp-changes</command>), recreate the branch from master, and replay the commits (<command>git am temp-changes</command>)</para>
+</note>
```

Comment:
Ah, yeah that's much easier. (and goes to show how much git I'm not familiar with :)) 

(force pushed fix)